lighthouse/src/main.rs

```diff
@@ -585,15 +592,14 @@ fn run<E: EthSpec>(
 
     // Log panics properly.
     {
-        let log = log.clone();
+        //let log = log.clone();
         std::panic::set_hook(Box::new(move |info| {
-            crit!(
-                log,
-                "Task panic. This is a bug!";
-                "location" => info.location().map(ToString::to_string),
-                "message" => info.payload().downcast_ref::<String>(),
-                "backtrace" => %Backtrace::capture(),
-                "advice" => "Please check above for a backtrace and notify the developers",
+            error!(
```

Comment:
though tracing doesnt expose a `crit` level, we might want to create a custom `crit` level? I believe we've discussed this internally previously, happy to hear others opinions about this